### PR TITLE
feat: update package-lock version on publish

### PR DESCRIPTION
**Pull request by maasencioh**

### CODE REVIEW

**Comment by targos** [comment]
you need to add the file in the `toAdd` array if it exists

**Comment by maasencioh** [comment]
talking about package-lock, should I remove yarn from this one?

File: src/server/controllers/repo-manager.js [hunk 1]
```diff
@@ -114,9 +114,14 @@ function*publish() {
         }
 
         yield git.writePkg(pkg);
+
+        if (yield git.checkPkgLock()) {
+            yield git.writePkgLock(version);
```
Comment (targos) [inline src/server/controllers/repo-manager.js]: instead of having two functions you can return a boolean from this one

**Comment by targos** [comment]
> should I remove yarn from this one?

no

**Comment by maasencioh** [comment]
there's no test, but I think that now should be ok

**Review by targos — APPROVED** [review]

**Comment by targos** [comment]
thanks

**Comment by maasencioh** [comment]
thank you :+1:

**Pull request merged**